tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -1035,7 +1035,14 @@ def leaderboard(self, data=None, extra_info=False, only_pareto_frontier=False, s
                     If A is a descendant of B, then B is an ancestor of A.
                     If this model is deleted, then all descendant models will no longer be able to infer on new data, and their 'can_infer' values will be False.
                     A model can only have descendant models whose 'stack_level' are higher than itself.
-
+        extra_metrics : list, default = None
+            A list of metrics to calculate scores for and include in the output DataFrame.
+            Only valid when `data` is specified. The scores refer to the scores on `data` (same data as used to calculate the `score_test` column).
+            This list can contain any values which would also be valid for `eval_metric` in predictor init.
+            For example, `extra_metrics=['accuracy', 'roc_auc', 'log_loss']` would be valid in binary classification.
+            This example would return 3 additional columns in the output DataFrame, whose column names match the names of the metrics.
+            Passing `extra_metrics=[predictor.eval_metric]` would return an extra column in the name of the eval metric that has identical values to `score_test`.
+            This also works with custom metrics. If passing an object instead of a string, the column name will be equal to the `.name` attribute of the object.
```

Comment:
maybe also clarify the negative metric issue here as well? People keep asking about this

Reply:
added